torchmdnet/models/tensornet.py

```diff
@@ -480,18 +510,39 @@ def forward(
             edge_index, edge_attr[..., 2, None, None], S, X.shape[0]
         )
         msg = Im + Am + Sm
+        
+        prefactor = 1 if self.addtional_methods is not None else torch.ones_like(msg, device=msg.device, dtype=msg.dtype)
```

Comment:
currently is 1 when additional_labels is not None. But if I set is as 1 and additional_labels is None, this will return the normal behavior of tensornet

Reply:
Yes, it implements an equivalent behavior, but you are allocating a really big tensor to just fill it with ones and multiply it elementwise, which is expensive. My point here is you should be able to just set it to 1 always. Or make it a size 1 tensor that you expand_as(whatever)